app/models/financial_transaction_class.rb

```diff
@@ -18,4 +22,10 @@ def display
       I18n.t('activerecord.attributes.financial_transaction.amount')
     end
   end
+
+  private
+
+  def update_balance_of_ordergroups
+    ordergroups.each { |og| og.update_balance! }
+  end
```

Comment:
i'm too, but we have no good way to do this otherwise, and it's a VERY unusual operation, so it shouldn't matter that much

Reply:
It's indeed unusual, but I'd be reluctant to make people wary of changing the name, for example.
Actually, when/why exactly does the balance need to be changed?

Reply:
if we ignore some `FinancialTransactionClass` the `account_balance` can change

Reply:
So that is: on create and destroy (when `ignore_for_account_balance` is `false`), and when `ignore_for_account_balance` changes, right?

What about not doing this when the name changes?
We can move it to a background job if it becomes too slow in practice for someone.

Reply:
IMHO getting the relevant data from the database to do a selective change is much more comlicated, than just doing it for all ordergroups. as already stated, this is something which nearly never happens and I don't see the benefit of adding complex logic for such a rare case

Reply:
`create` would be not required, since there are no `FinancialTransaction` where we need to apply it.